cdap-app-fabric-tests/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -81,23 +81,28 @@ public void testStatistics() throws Exception {
     ProgramId mapreduceProgram = WORKFLOW_APP.mr(mapreduceName);
     ProgramId sparkProgram = WORKFLOW_APP.spark(sparkName);
 
+    // Time from program starting to program running
+    int turnoverTime = 1;
```

Comment:
Since we have added a distinction between a program starting and a program running, when we use `setStart` in the tests to persist the state, I thought I would have a variable to mark the time between a program starting and a program running. 

This variable represents the time between a program starting and a program running.